panopticum/static/js/admin_formset_handlers.js

```diff
@@ -148,7 +150,15 @@ class RequirementsStorage {
 
             storage.groups.forEach(function(group, id) {
                 var cl = "row1";
-                var header = "<tr class='group-name'><td colspan=" + colspan + ">" + group.name + "</td></tr>";
+                var doc_link = "";
+
+                if (group.doc_link)
+                    doc_link = " (<a href='" + group.doc_link + "'>doc link</a>)";
```

Comment:
```javascript
doc_link = ` (<a href="${group.doc_link}">doc link</a>)`;
```

https://stackoverflow.com/questions/1408289/how-can-i-do-string-interpolation-in-javascript